Dockerfile

```diff
@@ -0,0 +1,54 @@
+FROM ubuntu:latest
```

Comment:
I'd be fine with `from rocker` to make this simpler.

Reply:
I did not have luck building grpc on debian sid due to openssl 1.1